lib/highLevelConsumer.js

```diff
@@ -523,11 +518,13 @@ HighLevelConsumer.prototype.offsetRequest = function (payloads, cb) {
  * @param {Client~failedToRegisterConsumerCallback} cb A function to call the consumer has been registered
  */
 HighLevelConsumer.prototype.registerConsumer = function (cb) {
-    this.client.zk.registerConsumer(this.options.groupId, this.id, this.payloads, function (err) {
+    var zk = this.client.zk;
+    var groupId = this.options.groupId;
+    this.client.zk.registerConsumer(groupId, this.id, this.payloads, function (err) {
```

Comment:
You could have used `zk` here or `this.client.zk` again in the next statement for consistency